README.md

```diff
@@ -1,11 +1,85 @@
+    - `Stage current file`
+    - `Unstage current file`
+- Commit
+    - `Commit all changes`: Only commits all changes without pushing
+    - `Commit all changes with specific message`: Same as above, but with a custom message
+    - `Commit staged`: Commits only files that have been staged
+    - `Commit staged with specific message`: Same as above, but with a custom message
+- Backup
+    - `Create Backup`: Commits all changes. If "Push on backup" setting is enabled, will also push the commit.
+    - `Create Backup with specific message`: Same as above, but with a custom message
+    - `Create backup and close`: Same as `Create Backup`, but if running on desktop, will close the Obsidian window. Will not exit Obsidian app on mobile.
+- Remote 
+    - `Push`
+    - `Pull`
+    - `Edit remotes`
+    - `Remove remote`
+    - `Clone an existing remote repo`: Opens dialog that will prompt for URL and authentication to clone a remote repo
+    - `Open file on GitHub`: Open the file view of the current file on GitHub in a browser window. Note: only works on desktop
+    - `Open file history on GitHub`: Open the file history of the current file on GitHub in a browser window. Note: only works on desktop
+- Local
+    - `Initialize a new repo`
+    - `Create new branch`
+    - `Delete branch`
+    - `CAUTION: Delete repository`
+- Source Control View
+    - `Open source control view`: Opens side pane displaying [Source control view](#sidebar-view)
+    - `Edit .gitignore`
+    - `Add file to .gitignore`: Add current file to .gitignore
+
+## Mobile
+
+### Installation
+
+#### Existing Repo
+
+Follow these instructions for setting up an Obsidian Vault on a mobile device that is already backed up in a remote git repository. 
+
+The instructions assume you are using GitHub, but can be extrapolated to other providers.
+
+1. Make sure any outstanding changes on all devices are pushed and reconciled with the remote repo.
+2. Install Obsidian for Android or iOS.
+3. Create a new vault (or point Obsidian to an empty directory). Do NOT select `Store in iCloud` if you are on iOS.
+4. (Recommended for security) [Create a personal access token on GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). Minimal permissions required are "Read access to metadata" and "Read and Write access to code and commit statuses" for the repo you are going to use.
+5. In Obsidian settings, enable community plugins. Browse plugins to install Obsidian Git.
+6. Enable Obsidian Git (on the same screen)
+7. Go to Options for the Obsidian Git plugin (bottom of main settings page, under Community Plugins section)
+8. Under the "Advanced" section, fill in the username on your git server and your password/personal access token. NOTE: If using a personal access token (recommended), use `__token__` as your username, not your actual GitHub username.
+9. Leave the relative path setting empty (for advanced users only).
```

Comment:
Instead of stating this in the README, the settings page itself should be more self-explanatory. So I think this line can be removed.

Reply:
Okay, I'll update this and push once the Settings is updated.